packages/dev-env/src/seed/client.ts

```diff
@@ -583,6 +583,19 @@ export class SeedClient<
     return this.verifications[by][subject]
   }
 
+  async unverify(by: string, subject: string) {
+    const verification = this.verifications[by][subject]
```

Comment:
In case we don't even know the verifier:
```suggestion
    const verification = this.verifications[by]?.[subject]
```